scheduling/job_scheduling.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+
+"""
+Author : Mohit Kumar
+Job Sequencing Problem implemented in python
+"""
+from collections import namedtuple
+
+
+class Scheduling:
+    def __init__(self, jobs):
+        self.jobs = jobs
+
+    def schedule(self, n, deadline):
```

Comment:
Python type hints?  doctests?  Avoid single-letter variable names especially for function parameters.  See CONTRIBUTING.md.